core/dropdowndiv.js

```diff
@@ -177,7 +177,7 @@ Blockly.DropDownDiv.setColour = function(backgroundColour, borderColour) {
 Blockly.DropDownDiv.showPositionedByBlock = function(owner, block,
       opt_onHide, opt_secondaryYOffset) {
   var scale = block.workspace.scale;
-  var bBox = block.getHeightWidth();
```

Comment:
Why was getHeightWidth giving the wrong values?

Does this solve the problem of the dropdown showing up below the entire block instead of below the field?